<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogConfigStorage</class>
 <widget class="QDialog" name="dialogConfigStorage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure storage settings - Metro</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="layoutDirectory">
     <item>
      <widget class="QLabel" name="labelDirectory">
       <property name="text">
        <string>Base directory</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editDirectory">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="upperLine">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutNumbering">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkNumbering">
       <property name="text">
        <string>Enable automatic numberung by adding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="editNumberingIncrease">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>50000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumberingAlpha">
       <property name="text">
        <string>per measurement and pad zeros to</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="editNumberingPadding">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelNumberingBeta">
       <property name="text">
        <string>decimals</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="spacerAutomaticNumbering">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>1</width>
         <height>1</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutIndicators">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkIndicators">
       <property name="text">
        <string>Enable indicator variables in storage name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelShowIndicators">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;[&lt;a href=&quot;#indicators&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;Show current&lt;/span&gt;&lt;/a&gt;]&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dialogConfigStorage</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkNumbering</sender>
   <signal>toggled(bool)</signal>
   <receiver>editNumberingIncrease</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>59</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>60</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkNumbering</sender>
   <signal>toggled(bool)</signal>
   <receiver>editNumberingPadding</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>505</x>
     <y>59</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
